models/repo_collaboration.go

```diff
@@ -186,9 +186,51 @@ func (repo *Repository) DeleteCollaboration(uid int64) (err error) {
 		return err
 	}
 
+	// Unassign a user from any issue (s)he has been assigned to in the
+	// repository
+	if err := repo.removeIssueAssignees(sess, uid); err != nil {
+		return err
+	}
+
 	return sess.Commit()
 }
 
+func (repo *Repository) removeIssueAssignees(e Engine, userID int64) error {
+
+	err := repo.GetOwner()
```

Comment:
Please move this into if below

Reply:
line 206 will reference `repo.Owner`, so here load the owner, I don't think it's a problem.

Reply:
The problem is you should use `repo.getOwner(e)`.